rpcs3/Emu/System.h

```diff
@@ -247,6 +248,19 @@ class Emulator final
 		return m_dir;
 	}
 
+	// String for GUI dialogs.
+	const std::string& GetUsr() const
+	{
+		return m_usr;
+	}
+
+	// u8 for cell.
+	// TODO: I'm ashamed of this - cast AND type conversion. Must replace.
+	const u32 GetUsrId() const
+	{
+		return static_cast<u32>(std::stoul(m_usr));
```

Comment:
There's no need for either the cast or the conversion, you can have it in a separate `u32` and only compute it once.

Reply:
You're thinking store it in memory as a pointer, after its read from config.yml?

Reply:
Yeah, simplest approach would be to have a separate `m_usr_id` and do the type conversion in the place where you assign to `m_usr` the val from the config.